tests/test_class_oci_image.py

```diff
@@ -0,0 +1,106 @@
+# -*- coding: utf-8 -*-
+#
```

Comment:
Small nit: This test needs to be added to `test_files_touched` under the `# test` and `# tern/classes` sections of the file.